modules/common/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@erento/nestjs-common",
-  "version": "1.2.2",
+  "version": "1.2.3",
```

Comment:
the version bump should be always separate PR... you never know who will change the code in the meantime... version change (and npm publish) can be directly committed to the code

don't forget to clean also lock in this PR

p.s.: it is good to be "academic" when it comes to versioning